.github/workflows/copilot_deploy.yml

```diff
@@ -74,3 +84,13 @@ jobs:
     - name: Copilot deploy
       run: |
         copilot svc deploy --env ${{ inputs.environment || 'test' }}
+  post_deploy_tests:
+    needs: copilot_build
+    secrets:
+      E2E_PAT: ${{secrets.E2E_PAT}}
+      GOV_NOTIFY_API_KEY: ${{secrets.GOV_NOTIFY_API_KEY}}
+    uses: communitiesuk/funding-service-design-workflows/.github/workflows/post-deploy.yml@FS-3337 #main
```

Comment:
This will be reverted in a future commit - the @ part can be removed entirely, as it will be using the 'parent' branch (aka main)